resources/summer.html

```diff
@@ -63,9 +63,9 @@
 
 <p>There are many paid internships out there if you are looking to gain research experience over the summer months. Don't know where to start? Here's a list of programs we're familiar with.</p>
 
-<h3>URECA Summer Program/Explorations in STEM</h3>
+<h3>URECA Summer Program/Explorations in STEM/Frances Velay Fellowship</h3>
 
-<p>The Undergraduate Research and Creative Activities (URECA) Department funds Stony Brook students doing full-time faculty mentored research over the summer. Applicants must seek out a faculty mentor prior to applying. There are two programs to choose from: Explorations in STEM, which is geared towards freshmen/sophomores and includes weekly professional development workshops, and URECA, which is geared towards experienced researchers. You can find a link to their website <a href="https://www.stonybrook.edu/commcms/ureca/summer/urecasummer.php">here.</a></p>
+<p>The Undergraduate Research and Creative Activities (URECA) Department funds Stony Brook students doing full-time faculty mentored research over the summer. Applicants must seek out a faculty mentor prior to applying. As of 2022, there are three programs to choose from: Explorations in STEM, which is geared towards freshmen/sophomores and includes weekly professional development workshops; URECA, which is geared towards experienced researchers; and Frances Velay Women and Science Research Fellowship, which is designed specifically for women STEM majors and heavily encouraged towards those with less research experience. You can find a link to their website <a href="https://www.stonybrook.edu/commcms/ureca/summer/urecasummer.php">here.</a></p>
```

Comment:
> which is designed specifically for women STEM majors and heavily encouraged towards those with less research experience.

I think it's more natural to say "which is designed specifically form women STEM majors, with those having less research experience heavily encouraged to apply"